diff --git a/include/eosio/vm/function_traits.hpp b/include/eosio/vm/function_traits.hpp
index 76a8818..a55d3d1 100644
--- a/include/eosio/vm/function_traits.hpp
+++ b/include/eosio/vm/function_traits.hpp
@@ -102,9 +102,9 @@ namespace eosio { namespace vm {
                                                                std::tuple<std::conditional_t<Decay, std::decay_t<Args>, Args>...>>;
       template <bool Decay, typename F>
       constexpr auto get_types(F&& fn) {
-         if constexpr (is_callable_v<decltype(fn)>)
-            return get_types<Decay>(&F::operator());
-         else
+         // if constexpr (is_callable_v<decltype(fn)>)
+         //    return get_types<Decay>(&F::operator());
+         // else
             return get_types<Decay>(fn);
       }
 
@@ -144,9 +144,9 @@ namespace eosio { namespace vm {
       constexpr auto parameters_from_impl(R(Cls::*)(Args...)const &&) ->  pack_from_t<N, Args...>;
       template <std::size_t N, typename F>
       constexpr auto parameters_from_impl(F&& fn) {
-         if constexpr (is_callable_v<decltype(fn)>)
-            return parameters_from_impl<N>(&F::operator());
-         else
+         // if constexpr (is_callable_v<decltype(fn)>)
+         //    return parameters_from_impl<N>(&F::operator());
+         // else
             return parameters_from_impl<N>(fn);
       }
 
